airflow/www/templates/airflow/dags.html

```diff
@@ -226,7 +226,7 @@
           }
         });
       });
-      $.getJSON("{{ url_for('api_experimental.latest_dag_runs') }}", function(data) {
+      $.getJSON("{{ url_for('api_experimental.dag_runs_latest') }}", function(data) {
```

Comment:
Does not have a dag_run_url field, and execution_date may not be formatted correctly, along with start_date processing not being there